src/datastore/postgres-store.ts

```diff
@@ -5135,44 +5174,51 @@ export class PgDataStore
       address,
       registered_at,
       expire_block,
-      zonefile_hash,
       zonefile,
+      zonefile_hash,
       namespace_id,
       tx_id,
       tx_index,
       status,
       canonical,
-      atch_resolved,
     } = bnsName;
-
-    await client.query(
-      `
-      INSERT INTO names(
-        name, address, registered_at, expire_block, zonefile_hash, zonefile, namespace_id,
-        tx_index, tx_id, status, canonical, atch_resolved,
-        index_block_hash, parent_index_block_hash, microblock_hash, microblock_sequence, microblock_canonical
-      ) values($1, $2, $3, $4, $5, $6, $7, $8,$9, $10, $11, $12, $13, $14, $15, $16, $17)
-      `,
-      [
-        name,
-        address,
-        registered_at,
-        expire_block,
-        zonefile_hash,
-        zonefile,
-        namespace_id,
-        tx_index,
-        hexToBuffer(tx_id),
-        status,
-        canonical,
-        atch_resolved,
-        hexToBuffer(blockData.index_block_hash),
-        hexToBuffer(blockData.parent_index_block_hash),
-        hexToBuffer(blockData.microblock_hash),
-        blockData.microblock_sequence,
-        blockData.microblock_canonical,
-      ]
-    );
+    // inserting remianing names information in names table
+    await this.queryTx(async client => {
```

Comment:
This should use the existing `client` passed to the function rather than opening a new sql tx.